traffic_portal/test/integration/PageObjects/ParametersPage.po.ts

```diff
@@ -124,4 +125,16 @@ export class ParametersPage extends BasePage {
     })
```

Comment:
Here: https://github.com/apache/trafficcontrol/blob/df56f95943e45e0355ac4537a67dadf8c7a9e9cf/traffic_portal/test/integration/PageObjects/ParametersPage.po.ts#L97 we should return only boolean. Read Brennan's explanation on why: https://github.com/apache/trafficcontrol/pull/5870#discussion_r635521989